rmgpy/rmgobject.pyx

```diff
@@ -115,3 +115,21 @@ cdef class RMGObject(object):
                 except ValueError:
                     pass
         self.__init__(**data)
+
+cpdef expand_to_dictionaries(obj):
```

Comment:
Maybe consider changing the name, since the function deals with more than just dicts

Reply:
Add a docstring?

Reply:
I like shorter names as long as they're still descriptive. Maybe just "expand_to_dict" or "repr_dict"?

Reply:
Another question, would there be any benefit to making this and `recursive_make_object` class methods of RMGObject?

Reply:
I'll refactor to `expand_to_dict`.

Reply:
As @mliu49 and I discussed offline, there would be benefit to making `expand_to_dict` and `recursive_make_object` static (as opposed to class) method of RMGObject, as we could call these functions from any class that inherits from RMGObject (meaning we would not have to import these functions explicitly most of the time). The downside of this is that these functions work on (hopefully) all objects, not just those that subclass RMGObject. Therefore, to avoid this confusion we will keep this as is